cores/esp8266/Schedule.h

```diff
@@ -39,6 +39,13 @@
 // scheduled function happen more often: every yield() (vs every loop()),
 // and time resolution is microsecond (vs millisecond). Details are below.
 
+// recurrent_max_grain_mS is used by delay() to let a chance to all
+// recurrent functions to accomplish their duty per their timing
+// requirement.
+// When it is 0, update_recurrent_grain() can be called to recalculate it.
```

Comment:
'let a chance for all' -> 'give a chance to all'
'accomplish their duty' -> 'complete'